### DISCUSSION TITLE
Odd values in map scale

### DISCUSSION THREAD

**Discussion by kds112** [devemux86/cruiser#288] [Q&A] [answered]
The map scale is always fixed and the value in meters is mostly odd like 140m or 210m or 730m. To estimate a distance on the map, a more even scale value would be helpful like 100m oder 200m or 5km. Therefore the length of the map scale should be adjusted to the best value when zooming in or out.
Is it a useful improvement in your opnion?

**Comment by devemux86** [comment, marked as answer]
@kds112 you probably use the "Advanced" option in "Settings | Map | Map scale".
It displays both map scale + map zoom and it cannot have variable width.

You can revert to the default "Normal" map scale option that only shows the scale
and has fixed values with variable width.

**Reply by kds112** [reply]
Yes, you are right! Problem solved! Thanks!!

**Reply by devemux86** [reply]
The current zoom is also shown next to the zoom buttons.

**Reply by kds112** [reply]
Why do we need the advanced map scale? Zoom factor and scale are shown in the normal Version. Could it be removed because of simplification reasons in the application? Like the advanced position marker?

**Reply by devemux86** [reply]
In the past it was requested that the zoom be displayed somewhere permanently all the time.

I do not know if we still need it?

**Reply by WolfgangZ42** [reply]
Why not? It's not really needed, I guess. But it doesn't bother anyone - or?
